### ISSUE TITLE
NFR: Language setting needs to be applied to all website

### ISSUE THREAD

**Issue by ZoeBeykirch** [hlxsites/vg-macktrucks-com#487]
Hi Team, 
Seems like all pages for VTNA & Mack has no language setting, "<html lang="en">" or "<html lang="en-us">", this affects accessibility (WCAG level A). Probably this is a generic template thing and thus easy to fix.

I did follow up on CrownPeak utilization - and happened to observe...
Pages are crawled by CrownPeak "https://dqm.crownpeak.com/group/accessibility", so anyone with access can do a quick check, please tell me if I should add users in CrownPeak for any walkthrough of all issues.

**Comment by ZoeBeykirch** [comment]
Confirmed this issue is still present. Language page source cannot be found in the most recent crown peak report. ![Image](https://github.com/user-attachments/assets/dea81744-409a-4c01-8a52-7c67d6486265)
